src/HasSlug.php

```diff
@@ -106,7 +106,7 @@ protected function hasCustomSlugBeenUsed(): bool
     {
         $slugField = $this->slugOptions->slugField;
 
-        return $this->getOriginal($slugField) != $this->$slugField;
+        return $this->getOriginal($slugField) != $this->$slugField && !empty($this->$slugField);
```

Comment:
This could be made more readable. Could you check this is a separate if statement?

Could you also add a test to make sure this works as intended?

Reply:
Will do 👍